agri-store-react/src/Components/Rating.js

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from "react";
+
+function Rating({ rating, onClick, style }) {
+  return (
+    <div>
+      {[...Array(5)].map((_, i) => (
+        <span key={i} onClick={() => onClick(i)} style={style}>
+          {rating > i ? (
+            <span
+              class="material-icons"
+              style={{ fontSize: "1.8rem", cursor: "pointer" }}
+            >
+              star
+            </span>
+          ) : (
+            <span
+              class="material-icons"
+              style={{ fontSize: "1.8rem", cursor: "pointer" }}
```

Comment:
avoid inline styles

Reply:
Thanks for the comment will remove that right now